app/build.gradle

```diff
@@ -52,14 +52,21 @@ android {
         timeOutInMs 10 * 60 * 1000  // 10 minutes
         installOptions "-d", "-t"
     }
+
+    productFlavors {
+      gms { }
```

Comment:
There should be some comments here explaining what each of the build variants are, and how they differ.